route/route.go

```diff
@@ -491,6 +492,7 @@ func (r *Router) batch(w http.ResponseWriter, req *http.Request) {
 			Data:        bev.Data,
 		}
 
+		addIncomingUserAgentFromRequest(ev, req)
```

Comment:
nit - Sadly, this is going to retrieve the same user-agent from the request header for all events. However, this does make the code cleaner

Reply:
Yeah, you're right. I might refactor to retrieve once and write once per event.

Reply:
Updated in [b901286](https://github.com/honeycombio/refinery/pull/1358/commits/b90128645022242e50cac6f97889fe62f504d5c7)